packages/elements/src/internals/machines/sign-up/router.machine.ts

```diff
@@ -156,6 +157,9 @@ export const SignUpRouterMachine = setup({
 
     isLoggedIn: or(['isStatusComplete', ({ context }) => Boolean(context.clerk.user)]),
     isSingleSessionMode: ({ context }) => Boolean(context.clerk?.__unstable__environment?.authConfig.singleSessionMode),
+    isRestricted: ({ context }) =>
+      context.clerk?.__unstable__environment?.userSettings.signUp.mode === SIGN_UP_MODES.RESTRICTED,
+    isRestrictedWithoutTicket: and(['isRestricted', not('hasTicket')]),
```

Comment:
@nikospapcom Is this affected by what we were discussing today ? Can you help @tmilewski make the change if necessary ?

Reply:
@tmilewski there is a case when the user has already accept the invitation and click the invitation link again
in this case we are displaying the following:
![Screenshot 2024-09-26 at 1 58 19 PM](https://github.com/user-attachments/assets/723036dc-3aa0-41eb-bf2a-41712910d5c3)
let's do a sync when you are available if you have any questions

Reply:
@tmilewski any update on that? do you need any help?